presto-docs/src/main/sphinx/admin/session-properties.rst

```diff
@@ -0,0 +1,25 @@
+==========================
+Session Properties Manager
+==========================
+
+Session properties manager provides a way to dynamically override default
+session properties. It supports overriding default session properties for each
+session based on ``SessionConfigurationContext``, which includes ``user``,
+``source`` and ``clientTags``. The returned properties override the default
+values but not the final values, thus any user-provided values will override
```

Comment:
The properties returned override the default values specified in code. However, session properties set by users are always respected, and take precedence over these values.

Reply:
The original text was the same as the java doc in code, which was reviewed by @electrum. I do think your wording is clearer though.